src/Task/RealMeSetupTask.php

```diff
@@ -91,8 +125,8 @@ private function validateInputs($forEnv)
             $errorList = PHP_EOL . ' - ' . join(PHP_EOL . ' - ', $this->errors);
 
             throw new Exception(_t(
-                'RealMeSetupTask.ERR_VALIDATION',
-                '',
+                self::class . '.ERR_VALIDATION',
+                'There were {numissues} issue(s) found during validation that must be fixed prior to setup: {issues}',
```

Comment:
I think you need to remove the empty string argument here. Should be `_t('ClassName.KEY', 'Default value {placeholder}', ['placeholder' => $dynamicValue])`